db/wide/db_wide_basic_test.cc

```diff
@@ -452,6 +473,31 @@ TEST_F(DBWideBasicTest, MultiCFMultiGetEntityAsPinnableAttributeGroups) {
         ASSERT_TRUE(results[i][j].status().IsInvalidArgument());
       }
     }
+    // Check for invalid column family in Attribute Group result
+    results.clear();
+    results.emplace_back(create_result(null_and_hot_cfs));
+    results.emplace_back(create_result(all_cfs));
+    db_->MultiGetEntity(ReadOptions(), num_keys, keys.data(), results.data());
+
+    // First one failed due to null CFs in the AttributeGroup
+    // Null CF
+    ASSERT_NOK(results[0][0].status());
```

Comment:
I wonder if checking both `NOK` and `IsInvalidArgument` is redundant, since it's a `Status` internal invariant.

Reply:
Actually it can be either `IsInvalidArgument` and `IsIncomplete` depending on the cfh. I needed to check the subcode as well.